packages/tabs/src/Tab.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export interface TabProps {
+  /**
+   * Value supplied to the data-tab-id attribute. Used to determine what Tab is active.
+   */
+  value: string;
```

Comment:
This is an implementation detail right? Can we make this optional? Do we really need to force the user to set this?

Reply:
Currently, It's how we're managing what Tab should be active, but happy to discuss alternatives to this approach